src/components/Filter.js

```diff
@@ -0,0 +1,75 @@
+// import * as json from "src/mock-data/tickets.json";
+import {useState} from "react";
+
+export function Filter() {
+    const [all, setAll] = useState(true);
+    const [noStops, setNoStops] = useState(false);
+    const [oneStop, setOneStop] = useState(false);
+    const [twoStops, setTwoStops] = useState(false);
+    const [threeStops, setThreeStops] = useState(false);
+    const [value, setValue] = useState("all");
+    // const tickets = json.default;
+
+    return (
+        <>
+            <div className="filter-stop">
+                <p className="title">КОЛИЧЕСТВО ПЕРЕСАДОК</p>
+
+                {/*All*/}
+                <div className="unit">
```

Comment:
Все дивы можно рендерить с помощью map для того, чтобы избежать дублирования кода
```suggestion
const checkboxes = [{name:  'none'}, {name: 'one'}];
const checkboxElements = checkboxes.map((element) => <div className="unit">{element.name}</div>
return <div>{checkboxElements}</div>
```